sentinelone/plugin.spec.yaml

```diff
@@ -1111,6 +1111,36 @@ actions:
         description: Number of entities affected by the requested operation
         type: integer
         required: false
+  blacklist:
```

Comment:
Or since we are blacklist/unblacklisting based on state should this instead be `set_hash_blacklist_state`

Reply:
Standardizing on `blackslist` here too across all EDR integrations.